External Renderer/Assets/Scripts/Camera/CustomCamera.cs

```diff
@@ -1,183 +1,178 @@
+        /// <summary>
+        /// Coroutine that renders an image every 
+        /// [<paramref name="delay"/>] seconds.
+        /// </summary>
+        /// <param name="renderSize">The resolution of the rendered image.</param>
+        /// <param name="delay">The interval between each render.</param>
+        /// <returns></returns>
+        private IEnumerator RendererCoroutine(Vector2Int renderSize, float delay)
         {
-            RenderImage(folder, renderSize);
-            yield return new WaitForSecondsRealtime(delay);
+            while (true)
+            {
+                RenderImage(renderSize);
+                yield return new WaitForSecondsRealtime(delay);
+            }
         }
-    }
 
-    /// <summary>
-    /// Render the view of the camera repeatedly.
-    /// </summary>
-    /// <param name="delay">The interval between each render.</param>
-    /// <param name="folder">Folder in which to save the render.</param>
-    /// <param name="renderSize">The resolution of the rendered image.</param>
-    public void StartIntervalRendering(float delay = 2f, string folder = "",
-        Vector2Int renderSize = default(Vector2Int))
-    {
-        _rendererCoroutine = 
-            StartCoroutine(RendererCoroutine(folder, renderSize, delay));
-    }
+        /// <summary>
+        /// Render the view of the camera repeatedly.
+        /// </summary>
+        /// <param name="delay">The interval between each render.</param>
+        /// <param name="renderSize">The resolution of the rendered image.</param>
+        public void StartIntervalRendering(float delay = 2f,
+            Vector2Int renderSize = default)
+        {
+            _rendererCoroutine =
+                StartCoroutine(RendererCoroutine(renderSize, delay));
+        }
 
-    /// <summary>
-    /// Stop rendering the view of the camera if StartIntervalRendering 
-    /// was called.
-    /// </summary>
-    public void StopIntervalRendering() {
-        if (_rendererCoroutine != null)
+        /// <summary>
+        /// Stop rendering the view of the camera if StartIntervalRendering 
+        /// was called.
+        /// </summary>
+        public void StopIntervalRendering()
         {
-            StopCoroutine(_rendererCoroutine);
-        } 
-        else
+            if (_rendererCoroutine != null)
+            {
+                StopCoroutine(_rendererCoroutine);
+            }
+            else
+            {
+                Debug.Log("No Screenshot Coroutine is active.");
+            }
+        }
+
+        // TODO Add multicam support.
+        /// <summary>
+        /// On runtime load, attach the customCamera to the first gameobject
+        /// found with a camera component.
+        /// </summary>
+        [RuntimeInitializeOnLoadMethod]
+        public static void AttachToCamera()
         {
-            Debug.Log("No Screenshot Coroutine is active.");
-        }   
-    }
+            Camera cam = FindObjectOfType<Camera>();

```

Comment:
For multicam support, this would be an array, and have `FindObjectsOfType<Camera>()` and loop over all and attach. if empty array, then write to log.